# Generated from: /home/latty/link_dir/2015_li3ds/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogDvl.h
# SbgLogDvlData
uint32 timeStamp
uint16 status
float32[3] velocity
float32[3] velocityStdDev